e2e_tests/tests/per_provider/normal_tests/import_key.rs

```diff
@@ -62,6 +67,21 @@ const KEY_PAIR_DATA: [u8; 609] = [
     0x27,
 ];
 
+#[cfg(feature = "cryptoauthlib-provider")]
+//#[allow(dead_code)]
+const PRIV_KEY_ECC: [u8; 32] = [
+    0xF5, 0xDB, 0x6B, 0xA1, 0x82, 0x22, 0xCE, 0xC1, 0x54, 0x53, 0xE5, 0x63, 0xDE, 0xC5, 0xC7, 0x94,
+    0xCD, 0x48, 0x95, 0xF2, 0x8C, 0xC2, 0x7F, 0x50, 0xC2, 0x7E, 0xC3, 0x1B, 0xAF, 0x44, 0xEA, 0x54,
+];
+#[cfg(feature = "cryptoauthlib-provider")]
+const PUB_KEY_ECC: [u8; 64] = [
```

Comment:
Just checking, is this format the expected one as described [here](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/psa_export_public_key.html)? If the Mbed Crypto provider supports importing public keys, would be nice to try on it as well to make sure!

Reply:
Oh well - it is not. I will correct the format. Many thanks for spotting.

Reply:
It was just corrected, I mean a proper DER format  is used. As noted before - since no support via picky-asn is available at the moment, the binary DER sequence is being parsed "manually" via openssl calls.